src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -80,6 +82,11 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
     const policyOwnerDisplayName = ownerDetails.displayName ?? policy?.owner ?? '';
     const companyCards = CardUtils.getMemberCards(policy, allCardsList, accountID);
 
+    // To render correctly transfer owner button
+    const [fundList] = useOnyx(ONYXKEYS.FUND_LIST, {initialValue: {}});
+    const isNative = getPlatform() === CONST.PLATFORM.IOS || getPlatform() === CONST.PLATFORM.ANDROID;
```

Comment:
We don't prefer to use getPlatform function in our App. We should create a new function called shouldShowTransferButton and specific logic for native and web separately